packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,450 @@
+  ]} />
+
+## Example
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu>
+    <Item>Aardvark</Item>
+    <Item>Kangaroo</Item>
+    <Item>Snake</Item>
+  </Menu>
+</MenuTrigger>
+```
+
+## Content
+
+The MenuTrigger accepts exactly two children: the Menu and the element which
+triggers the opening of the Menu. The trigger must be the first child passed into
+the MenuTrigger and should be an element that supports press events.
+
+If the Menu is open it will close on blur or scroll events.
+
+### Value
+
+The Menu's selected Item is propagated via the event [`onAction`](#onaction).
+
+The `defaultSelectedKeys` prop can be used to preselect Menu Items in the Menu
+placing it in an uncontrolled state. Alternatively, the `selectedKeys` preselect
+items in the Menu placing it in a controlled state.
+
+```tsx example
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Controlled
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    selectionMode="single"
+    selectedKeys={['Kangaroo']}>
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Uncontrolled
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    selectionMode="single"
+    defaultSelectedKeys={['Snake']}>
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+The `selectionMode` prop can be used to switch from the default selection of a single Menu Item to multiple Menu Items or none.
+
+```tsx example
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Selection Multiple
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    defaultSelectedKeys={['Kangaroo', 'Snake']}
+    selectionMode="multiple">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Selection None
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    selectionMode="none">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Sections
+
+Menus may have Sections with titles and children items.
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Section 1', children: [{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}, {name: 'Section 2', children: [{name: 'Koala'}]}]} itemKey="name">
+    {item => (
+      <Section items={item.children} title={item.name}>
+        {item => <Item>{item.name}</Item>}
+      </Section>
+    )}
+  </Menu>
+</MenuTrigger>
+```
+
+### Sematic Elements
+
+A Menu item's content may be any renderable node, not just strings.
```

Comment:
"Menu Item's" or "MenuItem's" to make it consistent with the rest of the doc

Reply:
I'm pretty sure I already made it consistent to "Menu Items".

Reply:
Ah then in this case it should be "A Menu Item's content" here right?